spec.yaml

```diff
@@ -109,6 +109,51 @@ paths:
             $ref: "definitions.yaml#/definitions/V4GenericResponse"
 
   /v4/clusters/:
+    get:
+      operationId: getClusters
+      tags:
+        - clusters
+      summary: Get clusters
+      description: |
+        This operation allows to fetch a list of clusters.
```

Comment:
@marians when I said I agree on fetches. I meant I agree with @JosephSalisbury ;) "this operation fetches" reads better to me.